### PR TITLE
SiteSetupList: add a unique key to each return statement

### PR DESCRIPTION
**Pull request by cpapazoglou**
#### Changes proposed in this Pull Request

* Add a unique key to each return statement by using `Fragment`
Attempts to fix the following console error

#### Testing instructions

* Check out this branch locally
* `yarn && yarn start`
* Create a new free site
* Go to https://wordpress.com/home for that site
* Open the console
* You should not get the following error

### CODE REVIEW

File: client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx [hunk 1]
```diff
@@ -233,7 +233,7 @@ const SiteSetupList = ( {
 					const isCompleted = task.isCompleted;
 
 					return (
-						<>
+						<Fragment key={ task.id }>
```
Comment (kwight) [inline client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx:236]: Hunh, didn't realize we could do that. https://reactjs.org/docs/fragments.html#keyed-fragments

**Comment by kwight** [comment]
I can't actually replicate the original console log, but I do see how your fix makes sense.

**Review by kwight — APPROVED** [review]
Fix looks good, even if I can't replicate ¯\_(ツ)_/¯

File: client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx (hunk 1 above)
Comment (cpapazoglou) [reply client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx:236]: Today I learned that <></> was a documented practise! I thought it was hack!

**Pull request merged**